timesketch/models/sketch.py

```diff
@@ -605,3 +607,73 @@ def __init__(
         self.user = user
         self.attribute = attribute
         self.value = value
+
+
+class Graph(LabelMixin, CommentMixin, BaseModel):
+    """Implements the graph model."""
+    user_id = Column(Integer, ForeignKey('user.id'))
+    sketch_id = Column(Integer, ForeignKey('sketch.id'))
+    name = Column(UnicodeText())
+    description = Column(UnicodeText())
+    graph_config = Column(UnicodeText())
+    graph_elements = Column(UnicodeText())
+    graph_thumbnail = Column(UnicodeText())
+    num_nodes = Column(Integer)
+    num_edges = Column(Integer)
+
+    def __init__(self, user, sketch, name, description=None, graph_config=None,
+                 graph_elements=None, graph_thumbnail=None, num_nodes=None,
+                 num_edges=None):
+        """Initialize the Graph object.
+
+        Args:
+            user (User): The user who created the graph.
+            sketch (Sketch): The sketch that the graph is bound to.
+            name (str): Name of the graph.
+            description (str): Description of the graph.
+            graph_config (dict): Config used when generating the graph.
+            graph_elements (str): Graph in json string format.
+            graph_thumbnail (str): Image of graph in Base64 format.
+            num_nodes (int): Number of nodes in the graph.
+            num_edges (int): Number of edges in the graph.
+        """
+        super(Graph, self).__init__()
```

Comment:
we should use the py3 way of using super.... 

super().__init__()

that is empty super calls (need to do that everywhere i the file)

Reply:
Noted, will refactor in a later PR :)